cloud-reactor-api/src/main/java/com/sequenceiq/cloudbreak/cloud/event/instance/RebootInstancesRequest.java

```diff
@@ -0,0 +1,31 @@
+package com.sequenceiq.cloudbreak.cloud.event.instance;
+
+import java.util.List;
+
+import com.sequenceiq.cloudbreak.cloud.context.CloudContext;
+import com.sequenceiq.cloudbreak.cloud.event.CloudPlatformRequest;
+import com.sequenceiq.cloudbreak.cloud.model.CloudCredential;
+import com.sequenceiq.cloudbreak.cloud.model.CloudInstance;
+
+public class RebootInstancesRequest<T> extends CloudPlatformRequest<T> {
+
+    private final List<CloudInstance> cloudInstances;
+
+    public RebootInstancesRequest(CloudContext cloudContext, CloudCredential cloudCredential,
+            List<CloudInstance> cloudInstances) {
+        super(cloudContext, cloudCredential);
+        this.cloudInstances = cloudInstances;
+    }
+
+    public List<CloudInstance> getCloudInstances() {
+        return cloudInstances;
+    }
+
+    @Override
+    public String toString() {
+        StringBuilder sb = new StringBuilder("RebootInstancesRequest{");
```

Comment:
Stringbuilder is overkill here I think. I know there is some class where it's implemented that way, but I don't think that's a good pattern. Also inherited classes should include `super.toString()` or we can loose useful data

Reply:
fixed